From db30b0b71225015826e830a3f678fdaae47272cc Mon Sep 17 00:00:00 2001
From: Michael Haubenwallner <michael.haubenwallner@ssi-schaefer.com>
Date: Tue, 21 May 2019 17:53:28 +0200
Subject: [PATCH 05/19] drop needless checks for libbsd and libelf

Don't check for libbsd and libelf, they aren't needed anywhere
and may break copying binaries from one linux to another, when
they did exist on the build-machine but not on the target.
---
 configure.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.in b/configure.in
index 0b0dc91..54ec42f 100644
--- a/configure.in
+++ b/configure.in
@@ -1285,9 +1285,7 @@ HAVE_X11=$have_x
 
 
 AC_CHECK_LIB(socket, open)
-AC_CHECK_LIB(bsd, open)
 
-AC_CHECK_LIB(elf, open)
 AC_CHECK_LIB(dl, open)
 AC_CHECK_LIB(dld, open)
 AC_CHECK_LIB(ld, open)
-- 
2.19.2

